apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -614,14 +725,14 @@ export const DispatchMetricInfo = ({
   // or if scoped to many markets then indicate it's scoped to "specific markets"
   if (marketNames.length > 1) {
     additionalDispatchMetricInfo = (
-      <Tooltip description={marketNames}>
+      <Tooltip description={marketNames.join(', ')}>
         <span>{t('Specific markets')}</span>
       </Tooltip>
     );
   }
 
   return (
-    <span data-testid="dispatch-metric-info">
+    <span data-testid="dispatch-metric-info" className="h-[2.5rem]">
```

Comment:
(non blocking): Is this added just for alignment? It will break with long text. It would probably be better to use subgrid, I did something similar in apps/trading/components/competitions/competitions-cta.tsx.

Reply:
`grid md:grid-rows-[subgrid] row-span-2` looks a bit off to me I will paste a screenshot

Reply:
<img width="1679" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/e39ae6b6-3e22-4082-94cd-efb765c32296">

Reply:
If you are happy with this I can push the change @mattrussell36

Reply:
No I'm not. We'll have to come back to this. This is fine for now